src/actions/_common-action-utils.js

```diff
@@ -0,0 +1,40 @@
+import path from 'path';
+import * as fspp from '../fs-promise-proxy';
+
+export const makeDestPath = (data, cfg, plop) =>
+	path.resolve(plop.getDestBasePath(), plop.renderString(cfg.path || '', data));
+
+export function* getTemplate(data, cfg, plop) {
+	const makeTmplPath = p => path.resolve(plop.getPlopfilePath(), p);
+
+	let { template } = cfg;
+
+	if (cfg.templateFile) {
+		const isDynamicTemplatePathAllowed = cfg.type !== 'addMany';
+		const templateFile = isDynamicTemplatePathAllowed
```

Comment:
@macrozone sorry it's taken so long for me to get to this. Do we the logic branch here? I believe addMany creates valid `add` action configs, so would it cause problems to have the `renderString()` method executed on those paths as well?

Reply:
@macrozone I was able to dig in a bit and figured out why you wrote it this way. I was able to refactor the `addMany` action a bit to make it work without this branch :tada:

Reply:
i just refactored the duplicated code out, so that the tests still work. I did not dig deeper in addMany